tests/integration/test_mask_sensitive_info_in_logs/test.py

```diff
@@ -0,0 +1,323 @@
+    ]
+
+    for encryption_function in encryption_functions:
+        node.query(f"SELECT {encryption_function}")
+
+    check_logs(
+        must_contain=[
+            "SELECT encrypt('aes-256-ofb', '[HIDDEN]')",
+            "SELECT encrypt('aes-256-gcm', '[HIDDEN]')",
+            "SELECT decrypt('aes-256-ofb', '[HIDDEN]')",
+            "SELECT aes_encrypt_mysql('aes-256-ofb', '[HIDDEN]')",
+            "SELECT aes_decrypt_mysql('aes-256-ofb', '[HIDDEN]')",
+            "SELECT tryDecrypt('aes-256-ofb', '[HIDDEN]')",
+        ],
+        must_not_contain=[plaintext, cipher, key, iv8, iv16, add],
+    )
+
+
+def test_create_dictionary():
+    password = new_password()
+
+    node.query(
+        f"CREATE DICTIONARY dict1 (n int DEFAULT 0, m int DEFAULT 1) PRIMARY KEY n "
+        f"SOURCE(CLICKHOUSE(HOST 'localhost' PORT 9000 USER 'user1' TABLE 'test' PASSWORD '{password}' DB 'default')) "
+        f"LIFETIME(MIN 0 MAX 10) LAYOUT(FLAT())"
+    )
+
+    check_logs(
+        must_contain=[
+            "CREATE DICTIONARY dict1 (`n` int DEFAULT 0, `m` int DEFAULT 1) PRIMARY KEY n "
+            "SOURCE(CLICKHOUSE(HOST 'localhost' PORT 9000 USER 'user1' TABLE 'test' PASSWORD '[HIDDEN]' DB 'default')) "
+            "LIFETIME(MIN 0 MAX 10) LAYOUT(FLAT())"
+        ],
+        must_not_contain=[password],
+    )
+
+    node.query("DROP DICTIONARY dict1")
+
+
+def test_backup_to_s3():
+    node.query("CREATE TABLE temptbl (x int) ENGINE=Log")
+    password = new_password()
+
+    queries = [
+        f"BACKUP TABLE temptbl TO S3('http://minio1:9001/root/data/backups/backup1', 'minio', '{password}')",
+        f"RESTORE TABLE temptbl AS temptbl2 FROM S3('http://minio1:9001/root/data/backups/backup1', 'minio', '{password}')",
+    ]
+
+    for query in queries:
+        # query_and_get_answer_with_error() is used here because we don't want to stop on error "Cannot connect to AWS".
+        # We test logging here and not actual work with AWS server.
+        node.query_and_get_answer_with_error(query)
+
+    check_logs(
+        must_contain=[
+            "BACKUP TABLE temptbl TO S3('http://minio1:9001/root/data/backups/backup1', 'minio', '[HIDDEN]')",
+            "RESTORE TABLE temptbl AS temptbl2 FROM S3('http://minio1:9001/root/data/backups/backup1', 'minio', '[HIDDEN]')",
+        ],
+        must_not_contain=[password],
+    )
+
+    node.query("DROP TABLE IF EXISTS temptbl")
+    node.query("DROP TABLE IF EXISTS temptbl2")
+
+
+def test_on_cluster():
+    password = new_password()
+
+    node.query(
+        f"CREATE TABLE table_oncl ON CLUSTER 'test_shard_localhost' (x int) ENGINE = MySQL('mysql57:3307', 'mysql_db', 'mysql_table', 'mysql_user', '{password}')"
+    )
+
+    check_logs(
+        must_contain=[
+            "CREATE TABLE table_oncl ON CLUSTER test_shard_localhost (`x` int) ENGINE = MySQL('mysql57:3307', 'mysql_db', 'mysql_table', 'mysql_user', '[HIDDEN]')",
+            "CREATE TABLE default.table_oncl",
```

Comment:
why do we check `CREATE TABLE default.table_oncl` only for this one ?

Reply:
I improved the test

Reply:
got it, thanks